pkg/simple/client/devops/jenkins/pipeline.go

```diff
@@ -661,54 +662,75 @@ func (p *Pipeline) CheckScriptCompile() (*devops.CheckScript, error) {
 }
 
 func (p *Pipeline) CheckCron() (*devops.CheckCronRes, error) {
-
 	var res = new(devops.CheckCronRes)
 
-	Url, err := url.Parse(p.Jenkins.Server + p.Path)
-
+	cronServiceURL, err := url.Parse(p.Jenkins.Server + p.Path)
```

Comment:
no error checking?

Reply:
I only change the variable naming nothing else. But consider no error checking is not a good practice, I will add error checking.

Reply:
fixed it at https://github.com/kubesphere/kubesphere/pull/3165/commits/730657166c4fbd577d2506a10bbbd9a38a3b9c1a